lib/mini_profiler_rails/railtie.rb

```diff
@@ -24,7 +24,12 @@ def self.initialize!(app)
 
     c.skip_paths ||= []
 
-    c.skip_paths << app.config.assets.prefix if serves_static_assets?(app)
+    begin
+      c.skip_paths << app.config.assets.prefix if serves_static_assets?(app)
+    rescue
+      # app.config did not repond_to? :assets
```

Comment:
I say we just fix that then, I am uneasy about hiding the error in a global
rescue

On Thu, Feb 4, 2016 at 3:08 PM, Ian Ker-Seymer <notifications@github.com>
wrote:

> In lib/mini_profiler_rails/railtie.rb
> <https://github.com/MiniProfiler/rack-mini-profiler/pull/201#discussion_r51827343>
> :
>
> > @@ -24,7 +24,12 @@ def self.initialize!(app)
> >
> >      c.skip_paths ||= []
> >
> > -    c.skip_paths << app.config.assets.prefix if serves_static_assets?(app)
> > +    begin
> > +      c.skip_paths << app.config.assets.prefix if serves_static_assets?(app)
> > +    rescue
> > +      # app.config did not repond_to? :assets
>
> first line is !app.respond_to(:assets)
>
> I noticed this as well, and was somewhat confused by it. Notice here we
> are checking if app.*config* responds to :assets, not just app. What is
> the reason for this?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/MiniProfiler/rack-mini-profiler/pull/201/files#r51827343>
> .
>